src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java

```diff
@@ -366,4 +373,36 @@ private String getDefaultSearchField(List<String> searchList) {
             return input.getSearchFields();
         }
     }
+
+    @Override
+    public Set<Cmdb> getCmdbByStaticAnalysis(String projectName) throws HygieiaException {
+        if (StringUtils.isNotEmpty(projectName)) {
+            Collector collector = collectorRepository.findByName("Sonar");
```

Comment:
instead of providing a default value of Sonar can you force the user to send it , if collector is not present for the name return back an error

Reply:
updated as requested.